### PR TITLE
bumped version

### PR DESCRIPTION
**Pull request by angellindo**

### CODE REVIEW

File: lib/phantomjs/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Phantomjs
-  VERSION = "1.9.8.0"
+  VERSION = "2.0.0"
```
Comment (BrianHawley) [inline lib/phantomjs/version.rb]: This won't work. It should be `"2.0.0.0"`.

File: lib/phantomjs/platform.rb [hunk 2]
```diff
@@ -114,7 +114,7 @@ def platform
         end
 
         def package_url
-          'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-i686.tar.bz2'
+          'https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.0.0-source.zip'
```
Comment (BrianHawley) [inline lib/phantomjs/platform.rb]: This gem doesn't have support for installing from source, and there will not be precompiled binaries made available for 2.0.0.

**Comment by BrianHawley** [comment]
This PR should be closed in favor of #66.

**Comment by colszowka** [comment]
:+1:

**Pull request closed without merging**